pkg/client/block/block.go

```diff
@@ -37,7 +38,8 @@ func newCometBlockEvent(blockMsgBz []byte) (client.Block, error) {
 
 	// If msg does not match the expected format then the block's height has a zero value.
 	if blockMsg.Block.Header.Height == 0 {
-		return nil, ErrUnmarshalBlockEvent.Wrap(string(blockMsgBz))
+		return nil, events.ErrEventsUnmarshalEvent.
+			Wrapf("unable to unmarshal block: %s", string(blockMsgBz))
```

Comment:
The error message is a bit redundant as `ErrEventsUnmarshalEvent` contains `"failed to unmarshal event bytes"` (similar to https://github.com/pokt-network/poktroll/pull/220#discussion_r1414581768).

```suggestion
		return nil, events.ErrEventsUnmarshalEvent.
			Wrapf("with block data: %s", string(blockMsgBz))
```